packages/core/src/hub.ts

```diff
@@ -188,6 +197,11 @@ export class Hub implements HubInterface {
     return this.getStackTop().scope;
   }
 
+  /** @inheritdoc */
+  public getIsolationScope(): Scope {
+    return this._isolationScope;
+  }
```

Comment:
Shouldn't this be a getter?

Reply:
we don't really have getters there right now, also this is really just a very short-lived thing anyhow and will be gone by v8 😅